src/parameters/AbstractAerosol.jl

```diff
@@ -0,0 +1,15 @@
+export AbstractAerosol
+
+"""
+    AbstractAerosol{FT}
+
+Struct of type AerosolType. Refers to an arbitrary aerosol.
+
+# Fields
+$(DocStringExtensions.FIELDS)
+"""
+Base.@kwdef struct AbstractAerosol{FT} <: AerosolType{FT} end
+
+function AbstractAerosol(FT)
```

Comment:
Probably better to have same form as the other aerosol types to keep a generic calibration code
i.e. `function AbstractAerosol(td::CP.AbstractTOMLDict)`. I just don't want to take up space in ClimaParams just to say `J_deposition_m_AbstractAerosol = FT(0)` though. Maybe there's a way to make a toml file within the function that says m and c values for ABDINM and ABIFM are 0...?